scripts/utils/sign_and_send.js

```diff
@@ -59,11 +54,9 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
       return
     }
 
-    assert(process.env.PK != null, "This script requires a private key be explicitly provided. Please export PK")
-    const privateKey = withHexPrefix(process.env.PK)
-    const account = web3.eth.accounts.privateKeyToAccount(privateKey)
-    console.log(`Signing and posting multi-send transaction ${transactionHash} from proposer account ${account.address}`)
-    const sigs = signHashWithPrivateKey(transactionHash, privateKey)
+    const signer = (await web3.eth.getAccounts())[0]
+    console.log(`Signing and posting multi-send transaction ${transactionHash} from proposer account ${signer}`)
+    const sigs = web3.eth.sign(transactionHash, signer)
```

Comment:
This needs an await doesn't it?

Reply:
OH man! Is that the problem? Yea, it does need await, but I don' think I forgot it in the other location where this functionality is tested. The new problem I am facing is a `require` statement in Gnosis Safe

```
// Hashes are automatically approved by the sender of the message or when they have been pre-approved via a separate transaction
require(msg.sender == currentOwner || approvedHashes[currentOwner][dataHash] != 0, "Hash has not been approved");
```

Reply:
```suggestion
    const sigs = await web3.eth.sign(transactionHash, signer)
```